postgres.go

```diff
@@ -215,8 +248,16 @@ func (db DBConf) baseBackupUnpack(sb s3Backend, keyPath, backupTar string) error
 
 	log.Debug("Data ready for unpacking")
 
-	key := getKey(keyPath)
-	r, err := newDecryptor(key, fr)
+	privateKey, err := getPrivateKey(privateKeyPath, c4ghPassword)
+	if err != nil {
+		log.Error("Could not retrieve private key")
+
+		return err
+	}
+
```

Comment:
```suggestion
	privateKey, err := getPrivateKey(privateKeyPath, c4ghPassword)
	if err != nil {
		return err
	}

```

No point of logging an error here since `err` will be propagated to main and logged as `fatal`

Reply:
same